<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>958</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift</family>
    <pointsize>14</pointsize>
    <underline>true</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.500136, x2:1, y2:0.466, stop:0 rgba(1, 33, 116, 255), stop:1 rgba(9, 92, 184, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.500136, x2:1, y2:0.466, stop:0 rgba(48, 48, 48, 255), stop:1 rgba(101, 101, 101, 255));</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Times new Roman</family>
             <pointsize>18</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white;
font: family &quot;Times new Roman&quot;;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Sistema de Control de Asistencias&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="reloj">
           <property name="font">
            <font>
             <family>Times new Roman</family>
             <pointsize>15</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white;
font: family &quot;Times new Roman&quot;;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Jueves 29 de Abril de 2022&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>8</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(172, 26, 45);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="cuadroAbajo">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>350</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>40</y>
         <width>180</width>
         <height>180</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>180</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>180</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/cct/biometrics.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Number_1">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>80</y>
         <width>80</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: none;
background-color: rgb(204, 204, 204);
border-radius: 3px;
}
QPushButton:hover{
background-color: rgb(171, 171, 171);
}v</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Number_2">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>80</y>
         <width>80</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: none;
background-color: rgb(204, 204, 204);
border-radius: 3px;
}
QPushButton:hover{
background-color: rgb(171, 171, 171);
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Number_3">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>80</y>
         <width>80</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: none;
background-color: rgb(204, 204, 204);
border-radius: 3px;
}
QPushButton:hover{
background-color: rgb(171, 171, 171);
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Number_4">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>140</y>
         <width>80</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: none;
background-color: rgb(204, 204, 204);
border-radius: 3px;
}
QPushButton:hover{
background-color: rgb(171, 171, 171);
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Number_5">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>140</y>
         <width>80</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: none;
background-color: rgb(204, 204, 204);
border-radius: 3px;
}
QPushButton:hover{
background-color: rgb(171, 171, 171);
}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Number_6">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>140</y>
         <width>80</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: none;
background-color: rgb(204, 204, 204);
border-radius: 3px;
}
QPushButton:hover{
background-color: rgb(171, 171, 171);
}</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Number_7">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>200</y>
         <width>80</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: none;
background-color: rgb(204, 204, 204);
border-radius: 3px;
}
QPushButton:hover{
background-color: rgb(171, 171, 171);
}</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Aceptar">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>260</y>
         <width>170</width>
         <height>50</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: none;
background-color: rgb(204, 204, 204);
border-radius: 3px;
}
QPushButton:hover{
background-color: rgb(171, 171, 171);
}</string>
       </property>
       <property name="text">
        <string>Acerptar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Number_8">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>200</y>
         <width>80</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: none;
background-color: rgb(204, 204, 204);
border-radius: 3px;
}
QPushButton:hover{
background-color: rgb(171, 171, 171);
}</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Number_9">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>200</y>
         <width>80</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: none;
background-color: rgb(204, 204, 204);
border-radius: 3px;
}
QPushButton:hover{
background-color: rgb(171, 171, 171);
}</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Number_0">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>260</y>
         <width>80</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: none;
background-color: rgb(204, 204, 204);
border-radius: 3px;
}
QPushButton:hover{
background-color: rgb(171, 171, 171);
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ID">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>20</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
font: 18pt;
border-radius: 5px;
}
QLineEdit:focus {
border: 2px solid green;
}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="borrar">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>20</y>
         <width>80</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: none;
background-color: rgb(204, 204, 204);
border-radius: 3px;
}
QPushButton:hover{
background-color: rgb(171, 171, 171);
}</string>
       </property>
       <property name="text">
        <string>Borrar</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="mensaje">
         <property name="styleSheet">
          <string notr="true">background: none;
color: white;
</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Esperando código&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="horarios">
         <property name="font">
          <font>
           <underline>true</underline>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background: none;
border: none;
color: white;</string>
         </property>
         <property name="text">
          <string>Click aquí para consultar horarios y/o faltas</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>10</y>
         <width>171</width>
         <height>100</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/cct/footer.png);
background: none;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Assets/udgLogo.qrc"/>
 </resources>
 <connections/>
</ui>
